configure

```diff
@@ -1528,6 +1530,13 @@ Optional Features:
                           ENABLED while external-lapack-for-subproblems and
                           lapack2flame MUST be DISABLED. (Disabled by
                           default.)
+  --enable-legacy-lapack  Compile and build into libflame the legacy f2c files
+                          of lapack (Version 3.5.0). If this option is not
```

Comment:
-> "of LAPACK (Version 3.5.0)"

And similar below - in text LAPACK should be all caps.

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/4/commits/959b92826d2a5e688f739ac96c83d73139158a39